package.json

```diff
@@ -176,5 +179,16 @@
 		"test-unit:watch": "npm run test-unit -- --watch",
 		"test-unit-php": "docker-compose run --rm wordpress_phpunit phpunit",
 		"test-unit-php-multisite": "docker-compose run -e WP_MULTISITE=1 --rm wordpress_phpunit phpunit"
+	},
+	"lint-staged": {
+		"packages/*/package.json": [
+			"wp-scripts lint-pkg-json"
+		],
+		"*.js": [
+			"eslint"
+		],
+		"{docs/{root-manifest.json,tool/*.js},packages/{*/src/{actions,selectors}.js,components/src/*/README.md}}": [
```

Comment:
I bet this should include all type of `README.md` files located inside `packages` folder.
We should also update when new `packages/*/README.md` is added.

Reply:
Is it overkill to run this on every commit? How long does this task take to run?

Reply:
I think it only generates `manifest.json` file. It's super fast.

Reply:
@ntwb It only runs if a file has changed that matches the patterns listed here. It is also super fast. 

@gziolo Agreed.